patches/components-permissions-BUILD.gn.patch

```diff
@@ -1,12 +1,12 @@
 diff --git a/components/permissions/BUILD.gn b/components/permissions/BUILD.gn
-index 799755985b575a7464897d0969bf383c386d267c..84620efdf6cfc864cc6942272965b42382685c61 100644
+index 799755985b575a7464897d0969bf383c386d267c..5d024c0b16026a644902ec22cc129b9b930df62d 100644
 --- a/components/permissions/BUILD.gn
 +++ b/components/permissions/BUILD.gn
 @@ -89,6 +89,7 @@ source_set("permissions") {
      "//ui/base",
      "//url",
    ]
-+  import("//brave/components/permissions/sources.gni") sources += brave_components_permissions_sources deps += brave_components_permissions_deps
++  import("//brave/components/permissions/sources.gni") sources += brave_components_permissions_sources deps += brave_components_permissions_deps if (is_android) { deps += [ "//brave/components/permissions/android:jni_headers" ] }
```

Comment:
just add `//brave/components/permissions/android:jni_headers` to `brave_components_permissions_deps` with an `is_android` guard. These patches are designed this way so we don't have to change them for things like this